### PR TITLE
fix: rework overview pane in schema browser

### PR DESCRIPTION
**Pull request by walborn**
I have deleted unnecessary components, because move this logic inside main component

### CODE REVIEW

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 1]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
-                                            ?.State
-                                    }
-                                />
-                            ),
-                        },
-                    ]}
-                />
-            ),
-        };
+        if (!currentSchemaData) return undefined;
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:140]: please add curly braces

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 2]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
-        if (!component) {
-            component = <InfoViewer info={[{label: createdAtLabel, value: createdAt}]} />;
+        // show PathVersion
+        overview.push({label: 'Version', value: PathVersion});
+
+        // show created time if it's not 0
+        if (isNumeric(CreateStep) && Number(CreateStep)) {
+            overview.push({
+                label: 'Created',
+                value: dateTimeParse(Number(CreateStep))?.format('YYYY-MM-DD HH:mm'),
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:165]: let's add helper in `src/utils` to format dates in one place. This helper will be used here and in `formatDateTime` function.

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 3]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
+        const {CreateStep, PathType, PathSubType, PathId, PathVersion} = currentSchemaData;
+
+        const overview: InfoViewerItem[] = [];
+
+        // for any schema type
+        overview.push({label: 'Type', value: PathType?.replace(/^EPathType/, '')});
 
-        let component =
-            currentSchemaData?.PathType && pathTypeToComponent[currentSchemaData.PathType]?.();
+        // show SubType if it's not EPathSubTypeEmpty
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:148]: lets remove obvious comments

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 4]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
-                />
-            ),
-        };
+        if (!currentSchemaData) return undefined;
+        const {CreateStep, PathType, PathSubType, PathId, PathVersion} = currentSchemaData;
+
+        const overview: InfoViewerItem[] = [];
+
+        // for any schema type
+        overview.push({label: 'Type', value: PathType?.replace(/^EPathType/, '')});
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:146]: Please add all labels with i18n

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 5]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
+            });
+        }
+
+        // ExtSubDomain: (database)
+        // show Paths (DomainDescription.PathsInside)
+        // show Shards (DomainDescription.ShardsInside)
+        if (PathType === EPathType.EPathTypeExtSubDomain) {
+            overview.push({
+                label: 'Paths count',
+                value: PathDescription?.DomainDescription?.PathsInside?.length,
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:202]: in the types `PathsInside` is a string, it seems that counting it's length is incorrect

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 6]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
+            });
+            overview.push({
+                label: 'Shards count',
+                value: PathDescription?.DomainDescription?.ShardsInside?.length,
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:206]: in the types ShardsInside is a string, it seems that counting it's length is incorrect

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 7]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
+            });
+        }
+        if (PathType === EPathType.EPathTypePersQueueGroup) {
+            const pqGroup = PathDescription?.PersQueueGroup;
+
+            overview.push({label: 'Partitions count', value: pqGroup?.Partitions?.length});
+
+            const value = pqGroup?.PQTabletConfig?.PartitionConfig?.LifetimeSeconds;
+            if (value) {
+                const hours = (value / HOUR_IN_SECONDS).toFixed(2);
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:233]: let's move this calculation to function

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx [hunk 8]
```diff
@@ -137,65 +137,114 @@ export function ObjectSummary({
...
     const renderObjectOverview = () => {
-        const startTimeInMilliseconds = Number(currentSchemaData?.CreateStep);
-        const createdAt = startTimeInMilliseconds
-            ? formatDateTime(startTimeInMilliseconds)
-            : 'unknown';
-        const createdAtLabel = 'Created At';
-
-        // verbose mapping to guarantee a correct render for new path types
-        // TS will error when a new type is added but not mapped here
-        const pathTypeToComponent: Record<EPathType, (() => React.ReactNode) | undefined> = {
```
Comment (Raubzeug) [inline src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:148]: What do you think about not to delete this mapping? I think it's rather useful when add a new type.
In the past it was a headache to remember all places where we need to add info about new type.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/Tenant/ObjectSummary/ObjectSummary.tsx (hunk 8 above)
Comment (walborn) [reply src/containers/Tenant/ObjectSummary/ObjectSummary.tsx:148]: For old variant we are creating a new component for any type separately. But I've rewrote the logic to have common info and adding addition info for certain type. 

May be it is possible to rewrite with `pathTypeToInfo` function with Record<>, but I don't understand the meaning of this

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**